packages/components/atoms/f-button/stories/RouterButton.stories.js

```diff
@@ -79,3 +51,31 @@ export const RouterLinkComponent = () => ({
 });
 
 RouterLinkComponent.storyName = 'Router Link';
+
+RouterLinkComponent.argTypes = {
+    ...SharedButtonArgTypes,
+    buttonType:
+    {
+        control: { type: 'select' },
+        options: ['primary', 'secondary', 'outline', 'ghost', 'link'],
+        description: 'Choose a Button Type',
+        defaultValue: 'primary'
+    },
+    buttonSize:
+    {
+        control: { type: 'select' },
+        options: ['xsmall', 'small', 'medium', 'large'],
+        description: 'Choose a Button Size',
+        defaultValue: 'medium'
+    },
+    hasIcon: { description: 'no control on this story' },
+    href: { description: 'no control on this story' },
+    isLoading: { description: 'no control on this story' },
```

Comment:
Are these necessary if they can't be used?

Reply:
I want to reuse the share control file so best compromised plus doesn't affect anything.